apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -215,6 +215,9 @@ type OpenTelemetryCollectorSpec struct {
 	// HostNetwork indicates if the pod should run in the host networking namespace.
 	// +optional
 	HostNetwork bool `json:"hostNetwork,omitempty"`
+	// ShareNetworkProcess indicates if the pod's containers should share process namespace.
+	// +optional
+	ShareNetworkProcess bool `json:"shareNetworkProcess,omitempty"`
```

Comment:
Why is this named `ShareNetworkProcess` and not just `ShareProcessNamespace`? We try to stick to Kubernetes core attribute names unless there's a good reason to deviate.

Reply:
honestly, have no idea why. most likely bad copy paste... let me fix it.

Reply:
done. and added scenario e2e for this use case. cheers